src/components/views/Waiter/WaiterContainer.js

```diff
@@ -0,0 +1,21 @@
+import { connect } from 'react-redux';
+import Waiter from './Waiter';
+import {
+  getAll,
+  fetchFromAPI,
+  getLoadingState,
+  changeTableStatusInAPI,
+} from '../../../redux/tablesRedux';
+
+const mapStateToProps = (state) => ({
+  tables: getAll(state),
+  loading: getLoadingState(state),
+});
+
+const mapDispatchToProps = (dispatch) => ({
+  fetchTables: () => dispatch(fetchFromAPI()),
+  changeTableStatus: (status, id) =>
+    dispatch(changeTableStatusInAPI(status, id)),
+});
```

Comment:
```suggestion
const mapDispatchToProps = {
  fetchTables: fetchFromAPI,
  changeTableStatus: changeTableStatusInAPI,
};
```
Jest to tzw *Object shorthand form* 
Więcej możesz poczytać tutaj https://react-redux.js.org/using-react-redux/connect-mapdispatch#two-forms-of-mapdispatchtoprops

Reply:
Done.